src/components/navbar/index.test.jsx

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from "vitest";
+import { handleLogout } from "./index";
+
+describe("Navigation", () => {
+  it("should clear local storage when handleLogout is called", () => {
+    localStorage.setItem("testKey", "testValue");
+
+    handleLogout();
+
+    expect(localStorage.length).toBe(0);
```

Comment:
This is a flaky test. What if there is additional functionality added to this component that add more properties to the localstorage. You have the correct assertion below and can delete measuring the length.